src/components/FindMovie/FindMovie.js

```diff
@@ -1,55 +1,119 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './FindMovie.scss';
+import PropTypes from 'prop-types';
 
 import { MovieCard } from '../MovieCard';
-import movies from '../../api/movies.json';
-
-export const FindMovie = () => (
-  <>
-    <form className="find-movie">
-      <div className="field">
-        <label className="label" htmlFor="movie-title">
-          Movie title
-        </label>
-
-        <div className="control">
-          <input
-            type="text"
-            id="movie-title"
-            placeholder="Enter a title to search"
-            className="input is-danger"
-          />
-        </div>
+import getMovieFromServer from '../../api';
 
-        <p className="help is-danger">
-          Can&apos;t find a movie with such a title
-        </p>
-      </div>
+export const FindMovie = ({ addMovie, movies }) => {
+  const [movie, setMovie] = useState(null);
+  const [Title, setTitle] = useState('');
+  const [hasMovie, setHasMovie] = useState(false);
```

Comment:
```suggestion
```
this variable is redundant. For conditional rendering you can use your `movie` variable